### PR TITLE
docs(card): simplified BgColor example

### PR DESCRIPTION
**Pull request by nainardev**
Copied from Alert Docs

### CODE REVIEW

File: www/src/examples/Card/BgColor.js [hunk 1]
```diff
@@ -1,97 +1,22 @@
...
+  'primary',
+  'secondary',
+  'success',
+  'danger',
+  'warning',
+  'info',
+  'light',
+  'dark'
+].map((variant, idx) => (
+  <><Card bg={variant} text={variant=='light' ? 'dark' : 'white'}  style={{ width: '18rem' }}>
```
Comment (Mudassar045) [inline www/src/examples/Card/BgColor.js]: here `idx` is unused variable, but it should be use as `key={idx}` for Card component. And for docs site, react-bootstrap is following some `strict` checks so you should replace `variant=='light'` with `variant==='light'`

**Comment by nainardev** [comment]
Updated :+1:

File: www/src/examples/Card/BgColor.js [hunk 2]
```diff
@@ -1,97 +1,22 @@
...
     <Card.Header>Header</Card.Header>
     <Card.Body>
-      <Card.Title>Primary Card Title</Card.Title>
+      <Card.Title>{variant} Card Title </Card.Title>
```
Comment (Mudassar045) [inline www/src/examples/Card/BgColor.js]: Take a look at `Primary` is in titled-case but `variant` will put the value here in lower-case.

**Comment by nainardev** [comment]
Yeah I tried my best. The old example had capitalisation better because it had manually typed.

If I implement capitalisation, the example would become verbose and would make it difficult to read

**Comment by nainardev** [comment]
Then why Alert docs has it?

**Comment by nainardev** [comment]
https://react-bootstrap.github.io/components/alerts

**Comment by Mudassar045** [comment]
> Then why Alert docs has it?

I looked into, that's old one I guess, I just shared my thoughts with you. I hope maintainers will look into your PR for approval.

**Comment by nainardev** [comment]
I thought you were a maintainer

**Comment by Mudassar045** [comment]
> I thought you were a maintainer

I'm OSS + clean code lover.

**Comment by nainardev** [comment]
You have 905 contributions. Must've been really busy contributing I guess. Also you portfolio looks minimalist and clean

**Comment by Mudassar045** [comment]
> You have 905 contributions. Must've been really busy contributing I guess. Also you portfolio looks minimalist and clean

Yeah, but often. Thanks, I'm working on portfolio. You can also get into and make contributions of any kind.

**Comment by nainardev** [comment]
Will open a new error-free PR

**Comment by jquense** [comment]
This looks fine, but can you fix the lint issues. Making the code blocks shorter is a win, even if it's a bit clever. We want folks to be able to absorb the entire example quickly without a lot of scrolling and this does that, as well as being consistent with our other examples

**Comment by nainardev** [comment]
Don't know why Netlify check failed

**Comment by jquense** [comment]
Looks like it's an unrelated error

**Comment by mxschmitt** [comment]
> Looks like it's an unrelated error

yep, restarted the build, then it was "fixed".

**Pull request merged**